server/tserver/src/main/java/org/apache/accumulo/tserver/ThriftClientHandler.java

```diff
@@ -239,21 +237,21 @@ public void loadFiles(TInfo tinfo, TCredentials credentials, long tid, String di
           path = ns.makeQualified(path);
           newFileMap.put(new TabletFile(path), mapping.getValue());
         }
-        var files = newFileMap.keySet();
-        bulkImportStatus.updateBulkImportStatus(files, BulkImportState.INITIAL);
+        var fileStream = newFileMap.keySet().stream().map(TabletFile::getPathStr);
+        List<String> files = fileStream.collect(Collectors.toList());
```

Comment:
You might be able to get a oneline with a static import of `Collectors.toList()`, if that's what you're going for:

```suggestion
        var files = newFileMap.keySet().stream().map(TabletFile::getPathStr).collect(toList());
```